raven/event.go

```diff
@@ -8,9 +8,12 @@ import (
 	"runtime"
 	"strings"
 
+	"github.com/yext/glog"
 	"github.com/yext/glog-contrib/stacktrace"
 )
 
+const DSN_KEY = "dsn"
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps

Also, there's no reason to export this, I think?